src/katcp_fpga.py

```diff
@@ -537,4 +537,17 @@ def unhandled_inform(self, msg):
         if self.unhandled_inform_handler is not None:
             self.unhandled_inform_handler(msg)
 
+    def check_phy_counter(self):
+        request_args = ((0, 0), (0, 1), (1, 0), (1, 1))
```

Comment:
Something just occured to me -- are we not hardcoding the assumption that there are always exactly 4 interfaces per roach?